modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/Conjunction.es.js

```diff
@@ -38,6 +38,15 @@ function Conjunction({
 		className
 	);
 
+	const [activeLabel, setActiveLabel] = useState(null);
```

Comment:
good naming? means `activeConjunction`?

Reply:
is the `activeConjunctionLabel`, but given we're in the `Conjunction` component and is an internal state variable... I'll call it an `activeLabel`